.semaphore/release/hashrelease.yml

```diff
@@ -35,10 +28,10 @@ blocks:
             - ./bin/release hashrelease publish
       prologue:
         commands:
-          - cache restore release-${SEMAPHORE_GIT_SHA}
           - cd release
+          - cache restore release-${SEMAPHORE_GIT_SHA}
       env_vars:
         - name: OPERATOR_BRANCH
           value: master
         - name: IS_HASHRELEASE
```

Comment:
Do we still need this env var?

Reply:
It is still being used, although the default in config is set to true, I want to have it explictly called out here too

Reply:
Gotcha. I will probably remove this in my PR since it's redundant with the subcommand split.